code/modules/atmospherics/machinery/pipes/multiz.dm

```diff
@@ -32,26 +32,24 @@
 /obj/machinery/atmospherics/pipe/multiz/SetInitDirections()
 	initialize_directions = dir
 
-/obj/machinery/atmospherics/pipe/multiz/update_layer()
-	return // Noop because we're moving this to /obj/machinery/atmospherics/pipe
-
 /obj/machinery/atmospherics/pipe/multiz/update_overlays()
 	. = ..()
+	cut_overlays()
 	pipe.color = front_node ? front_node.pipe_color : rgb(255, 255, 255)
 	pipe.icon_state = "pipe-[piping_layer]"
 	. += pipe
 	center.pixel_x = PIPING_LAYER_P_X * (piping_layer - PIPING_LAYER_DEFAULT)
 	. += center
-
+	update_layer()
 
```

Comment:
Isn't this unrelated to the issues here?

Reply:
ah i forgot i added this, the multiz adapters have the same issue as manifolds where the overlay would "shadow" in other layers, should i remove it or just update the PR desc?

Reply:
I've got the same worries about this as your other pr, yeet it for now

Reply:
yeeted